tests/model/test_base_project.py

```diff
@@ -631,30 +632,183 @@ def dummy_conveyor_project():
         organization=BaseOrganization(team_list, workplace_list),
     )
     return project
-def test_component_place_check(dummy_conveyor_project):
+def test_component_place_check_1(dummy_conveyor_project):
```

Comment:
The name of this function is not good.
The name should be set so that we can understand what type of check is performed for this function.